okonomiyaki/file_formats/_egg_info.py

```diff
@@ -171,36 +179,32 @@ def _translate_invalid_requirement(s):
 
 
 def text_attr(**kw):
-    """ An attrs.attr-like descriptor to describe fields that must be unicode
-    but accept bytes/ascii (which are automatically converted).
+    """ An attrs.attr-like descriptor to describe fields that must be unicode.
     """
-    for k in ("validator", "convert"):
+    for k in ("validator", ):
         if k in kw:
             raise ValueError("Cannot pass '{0}' argument".format(k))
```

Comment:
yes, it is annoying for testing because `!r` becomes `<value>` on python 3 and `u<value>` on python 2.